omero/developers/Modules/Search.rst

```diff
@@ -57,10 +66,21 @@ FullTextAnalyzer <src/main/java/ome/services/fulltext/FullTextAnalyzer.java>`.
 
 Assuming these entries above for Image.name:
 
--  searching for **GFP-H2B** returns 1 and 2.
--  searching for **"GFP H2B"** also returns 1 and 2.
--  searching for **GFP H2B** returns 1, 2, and 3, since the two terms
+-  searching for **GFP-H2B** returns 1, 2, 3 and 4, because of the tokenizing on the hyphen and joining of the tokens by **OR**.
+This behavior is changed by adding wildcards or quotations, see below.
+-  searching for **"GFP H2B"** only returns 1 and 2, because the quotes enforces the exact sequence of the terms.
+-  searching for **GFP H2B** returns 1, 2, 3 and 4, since the two terms
    are joined by an **OR**.
+-  searching for **"GFP-H2B"** returns 1 and 2.
+
+With the same entries as above and adding a wildcard:
+
+-  searching for **"GF\*"** returns no results.
```

Comment:
This one surprises me a bit. Perhaps the quotes go a step further than just dropping syntax like `-` and `_` and even disable the `*`.

Reply:
Double-checked the behaviour. It is as specified.

Reply:
Verified all queries also in OMERO.insight. No difference with OMERO.web.

Reply:
@joshmoore I found https://stackoverflow.com/questions/35797085/how-does-lucene-work-with-quotes-and-wildcards - according to that answer, the ``"GF*"`` is interpreted as if it would be ``GF`` which does not igve any results in our case. The wording is that
`` Again, phrase query. Not a wildcard query in any way. You can't use wildcards in phrases. The analyzer will remove the punctuation, making this identical to the last one.``
Happy for me to use this wording, warning the user off using wildcards in phrases and move on here ?